### PR TITLE
Create shop section

### PR DESCRIPTION
**Pull request by IvanSidorenkov**

### CODE REVIEW

File: src/layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactNode } from 'react';
+
+import Header from "../layout/header/Header";
+import styles from "./Layout.module.scss";
```
Comment (AndreiLavrov) [inline src/layout/Layout.tsx:4]: "

File: src/layout/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+interface LayoutProps {
+  children: ReactNode;
+}
+
+const Layout: FC<LayoutProps> = ({ children }) => {
+  return (
+    <section className={styles.section}>
+      <Header />
+      <main>{children}</main>
+      {/*<Footer />*/}
```
Comment (AndreiLavrov) [inline src/layout/Layout.tsx:15]: better not to add comment at all for now?

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -1,15 +1,13 @@
 import type { AppProps } from 'next/app';
 
 import '../public/styles/reset.scss';
-import styles from './_app.module.scss';
-import Header from "../layout/header/Header";
+import Layout from "../layout/Layout";
```
Comment (AndreiLavrov) [inline src/pages/_app.tsx:4]: I guess some one of you has wrong settings for double quotes or it was just copied badly?

File: src/constants/mockData.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  {
+    category: 'обувь',
+    image: ''
+  },
+  {
+    category: 'площадка',
+    image: ''
+  },
+]
```
Comment (ssavitski) [inline src/constants/mockData.ts:30]: `;` is missed. eslint has to track such stuff

File: src/layout/Layout.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactNode } from 'react';
+
+import Header from "../layout/header/Header";
+import styles from "./Layout.module.scss";
+
+interface LayoutProps {
```
Comment (ssavitski) [inline src/layout/Layout.tsx:6]: start interfaces with letter `I`

File: src/layout/header/Header.module.scss [hunk 6]
```diff
@@ -25,7 +25,7 @@
 }
 
 .link {
-  font-family: 'Inter', sans-serif;
+  font-family: $inter;
```
Comment (ssavitski) [inline src/layout/header/Header.module.scss:28]: how many font families do we have?
maybe better to name it as font-primary or font-face-primary, font-face-secondary, etc.

File: src/layout/header/Header.module.scss [hunk 7]
```diff
@@ -25,7 +25,7 @@
...
   font-style: normal;
   font-weight: 400;
```
Comment (ssavitski) [inline src/layout/header/Header.module.scss:30]: put font weight values into variables

File: src/sections/delivery/Delivery.tsx [hunk 8]
```diff
@@ -2,17 +2,17 @@ import Image from 'next/image';
 
 import { useQuery } from '@/hooks/use-query';
 import styles from './Delivery.module.scss';
-import routeImg from '../../public/assets/images/route.png';
-import couponImg from '../../public/assets/images/routeCoupon.png';
-import starIcon from '../../public/assets/images/starIcon.png';
-import headPhonesImg from '../../public/assets/images/headPhonesBackground.png';
-import sneakerImg from '../../public/assets/images/sneakerBackground.png';
+import routeImg from '../../public/assets/images/png/route.png';
```
Comment (ssavitski) [inline src/sections/delivery/Delivery.tsx:5]: alias

File: src/sections/shops/Shops.module.scss [hunk 9]
```diff
@@ -0,0 +1,130 @@
+@import "src/public/styles/variables";
+
+@mixin shopCard {
```
Comment (ssavitski) [inline src/sections/shops/Shops.module.scss:3]: better to store mixins separately

File: src/sections/shops/Shops.tsx [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+const Shops = () => {
+  return (
+    <section className={styles.shopsSection}>
+      <h2 className={styles.title}>Доставляем ото всюду</h2>
+      <div className={styles.shopsWrapper}>
+        {SHOPS.map((shop) =>
+          <Shop key={shop.image} label={shop.category} image={shop.image} />
+        )}
+        <div>
+          <div className={styles.allShops}>
```
Comment (ssavitski) [inline src/sections/shops/Shops.tsx:17]: do we need 2 divs, couldn't we do what we need with one?

Comment (IvanSidorenkov) [reply src/sections/shops/Shops.tsx:17]: Yes, we need. I made component with it

**Review by ssavitski — APPROVED** [review]

**Pull request merged**